tests/phpunit/tests/media.php

```diff
@@ -1143,7 +1143,7 @@ public function test_add_image_size() {
 		remove_image_size( 'test-size' );
 
 		$this->assertArrayNotHasKey( 'test-size', $_wp_additional_image_sizes );
-		add_image_size( 'test-size', 200, 600 );
+		add_image_size( 'test-size', 200, 600, false, true );
```

Comment:
Same as above applies throughout this file.

Reply:
removed these changes